setup.cfg

```diff
@@ -9,6 +9,7 @@ addopts =
     --disable-pytest-warnings
     -rs
 
+
```

Comment:
why?

Reply:
You mean the blanc line? I think merge mistake